### DISCUSSION TITLE
Folders for organizing your Scheduled and Trigger tasks

### DISCUSSION THREAD

**Discussion by henrik123098** [gameserverapp/Platform#134] [Suggestions]
Folders for organizing your Scheduled and Trigger tasks on the GSA dashboard

The reason it would be nice to be able to organize your tasks is if, for example, you have a lot of event maps.

let's say you have 3-5 event maps that each have its own purpose.

and you want to make complicated tasks that all is co depended.

so if you have 5 maps and each has 10 tasks it gets very crowded very fast. so I believe being able to make subfolders that you can use to organize tasks would be a huge quality of life change.

another use case for this is the idea of using complex tasks to make quests with rewards. 

lets say you want to make 5 quests and each quest has 10 tasks where they need to do 1 task to unlock the next and so on.
then in this scenario, you also end up with 50 tasks which would have been great to be able to organize into 5 folders. 

now imagine doing both then you have 100 tasks good luck keeping track of what's what especially if you are more than 1 person making and using them.

**Comment by mvaessen** [comment]
Created development item #211
